src/client/Client.js

```diff
@@ -358,6 +358,17 @@ class Client extends BaseClient {
       .then(app => new ClientApplication(this, app));
   }
 
+  /**
+   * Obtains the widget from a guild, available for all guilds
+   * @param {GuildResolvable} guild The guild to fetch the widget for
+   * @returns {Promise<Object>}
```

Comment:
Don't we have a GuildWidget type/class? Shouldn't we return that instead?

Reply:
we have a "GuildWidget" typedef but its actually the Guild Widget Settings object, and we don't have a correct Guild Widget

Reply:
Done, I have create a correct Guild Widget (I think)